From a4c9c1950185d22a254cf4516d5b53d33e2b5c8c Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Tue, 11 Aug 2015 16:17:40 +0200
Subject: [PATCH] ccpp: fix comment related to 'MakeCompatCore' option in
 CCpp.conf

The comment fits only on the default core_pattern template.
If the core_pattern is not default, the comment does not fit.

Related to rhbz#1252384

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>
---
 src/hooks/CCpp.conf | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/src/hooks/CCpp.conf b/src/hooks/CCpp.conf
index d199116..f9bdd70 100644
--- a/src/hooks/CCpp.conf
+++ b/src/hooks/CCpp.conf
@@ -1,7 +1,12 @@
 # Configuration file for CCpp hook
 
-# If you also want to dump file named "core"
-# in crashed process' current dir, set to "yes"
+# CCpp hook writes its template to the "/proc/sys/kernel/core_pattern" file
+# and stores the original template in the "/var/run/abrt/saved_core_pattern"
+# file. If you want CCpp hook to create a core dump file named according to
+# the original template as well, set 'MakeCompatCore' to 'yes'.
+# If the original template string starts with "|", the string "core" is used
+# instead of the template.
+# For more information about naming core dump files see 'man 5 core'.
 MakeCompatCore = yes
 
 # Do you want a copy of crashed binary be saved?
-- 
2.6.3

